dotcom-rendering/src/components/FootballMatchesPageWrapper.importable.tsx

```diff
@@ -1,11 +1,63 @@
-import type {
-	FootballMatches,
-	FootballMatchKind,
-	Regions,
+import { isObject, isString, isUndefined } from '@guardian/libs';
+import type { Dispatch, SetStateAction } from 'react';
+import { useState } from 'react';
+import type { FEFootballDataPage } from '../feFootballDataPage';
+import {
+	type FootballMatches,
+	type FootballMatchKind,
+	getParserErrorMessage,
+	parse,
+	type Regions,
 } from '../footballMatches';
 import type { EditionId } from '../lib/edition';
+import type { Result } from '../lib/result';
+import { error, ok } from '../lib/result';
 import { FootballMatchesPage } from './FootballMatchesPage';
 
+export const getMoreDays =
+	(
+		nextPage: string,
+		setNextPage: Dispatch<SetStateAction<string | undefined>>,
+	) =>
+	async (): Promise<Result<'failed', FootballMatches>> => {
+		try {
+			const fetchResponse = await fetch(
+				'/MoreFootballData?nextPage=' + nextPage,
+			);
```

Comment:
I may be missing something, but could you please elaborate on why we need to have a new DCR endpoint for this? Could we not just make the api call to frontend json endpoint and get the json response? e.g. https://api.nextgen.guardianapps.co.uk/football/fixtures/more/2025/Mar/13.json?dcr

Reply:
I think you're right. I was initially thinking we would want to parse it on the backend but I ended up not doing that